tests/chainer_tests/functions_tests/loss_tests/test_sigmoid_cross_entropy.py

```diff
@@ -206,7 +208,7 @@ def forward(self):
     def test_call_cudnn_backward(self):
         with chainer.using_config('use_cudnn', self.use_cudnn):
             y = self.forward()
-            patch = 'cupy.cuda.cudnn.activationForward_v4'
+            patch = 'cupy.cudnn.activation_forward'
```

Comment:
Why is this change needed?

Reply:
Cython behinds `activationForward_v4` call .

Reply:
OK.